projects/libexif/exif_loader_fuzzer.cc

```diff
@@ -12,6 +12,35 @@ void data_func(ExifContent *content, void *user_data) {
   exif_content_foreach_entry(content, content_func, NULL);
 }
 
+static int
+test_exif_data (ExifData *d) {
+        unsigned int i, c;
+        char v[1024], *p;
+        char buf[2048];
+        ExifMnoteData *md;
+
+        md = exif_data_get_mnote_data (d);
+        if (!md)
+                return 1;
+
+        exif_mnote_data_ref (md);
+        exif_mnote_data_unref (md);
+
+        c = exif_mnote_data_count (md);
+        for (i = 0; i < c; i++) {
+                const char *name = exif_mnote_data_get_name (md, i);
+                if (!name) break;
```

Comment:
Could we use braces everywhere please? We should also expand the corpus to contain  samples that will trigger these paths if we don’t already have them.

Reply:
I added braces on these.

Not sure where the corpus is kept, the referenced samples https://github.com/ianare/exif-samples/tree/master/jpg contain maker notes at least.